src/Infrastructure/Common/ContainerBuilder.php

```diff
@@ -65,6 +68,7 @@ public static function getContainer(): Container
         $loader->load('services.yml');
         $loader->load('parameters.yml');
 
+        self::cleanVarDirecotry();
```

Comment:
Isn't it weird to clean a directory during the instantiation of the container? Furthermore, in this case, you clean the directory for all command launched including those which are not part of the migrations. Why not in the MigrationTool command instead? :)